content/docs/concepts/performance-and-scalability/index.md

```diff
@@ -13,127 +13,106 @@ aliases:
 
-<!-- add blueperf and more details -->
+## Control plane performance
 
-## Synthetic end to end benchmarks
+Pilot configures sidecar proxies based on user authored configuration and the current
+ state of the system. In a Kubernetes environment services, deployments, and endpoints
+ are the ambient configuration and state. Istio configuration like `gateway` and
+ `virtual service` is the user authored portion of the configuration.
+ Pilot processes the combined configuration and state to produce configuration for the proxies.
 
-We use [Fortio](https://fortio.org/) as Istio's synthetic end to end load testing tool. Fortio runs at a specified query per second (qps) and records an histogram of execution time and calculates percentiles (e.g. p99 i.e. the response time such as 99% of the requests take less than that number (in seconds, SI unit)). It can run for a set duration, for a fixed number of calls, or until interrupted (at a constant target QPS, or max speed/load per connection/thread).
+Control plane supports thousands of services, spread across thousands of pods with a
+ similar number of user authored virtual services and other configuration.
+ CPU and memory requirements for Pilot scale with the amount of configuration and state.
+ The CPU consumption also scales with the rate of change of deployments and the
+ rate of change of configuration.
 
-Fortio is a fast, small, reusable, embeddable go library as well as a command line tool and server process, the server includes a simple web UI and graphical representation of the results (both a single latency graph and a multiple results comparative min, max, average and percentiles graphs).
+Lastly, the number of proxies connecting to a pilot affect the pilot CPU,
+ however this part is inherently horizontally scalable.
 
-Fortio is also 100% open-source and with no external dependencies beside go and gRPC so you can reproduce all our results easily and add your own variants or scenarios you are interested in exploring.
+When [namespace isolation](/docs/reference/config/networking/v1alpha3/sidecar/) is enabled,
+ a single pilot instance can support 1000 services, 2000 sidecars with 1 vCPU and 1.5 GB of memory.
+ You may increase the number of pilots to reduce the amount of time it takes for configuration
+ to reach all the proxies.
 
-Here is an example of scenario (one out of the 8 scenarios we run for every build) result graphing the latency distribution for `istio-0.7.1` at 400 Query-Per-Second (qps) between 2 services inside the mesh (with mutual TLS, Mixer policy checks and telemetry collection):
+## Release 1.1 performance summary
 
-<iframe src="https://fortio.istio.io/browse?url=qps_400-s1_to_s2-0.7.1-2018-04-05-22-06.json&xMax=105&yLog=true" width="100%" height="1024" scrolling="no" frameborder="0"></iframe>
+**Mesh Traffic:** The total traffic in the mesh; includes ingress, egress, and all traffic going
```

Comment:
Why are you including this definition here? This seems entirely out of place. I would include it as part of the intro.